dangerfile.js

```diff
@@ -30,27 +31,28 @@ const fileChanges = [...modifiedFiles, ...createdFiles];
 // Always be nice.
 message('Thanks for the PR 🎉.');
 
+// Ensure new web components are extensible by consumers.
 const webComponentIntroduced = createdFiles.some(filePath => (
   filePath.match(/packages\/bpk-component.+\/src\/.+\.js/)
 ));
 
 if (webComponentIntroduced) {
-  message('It looks like you are introducing a web component');
-  warn('Ensure the component style is extensible via `className`');
+  message('It looks like you are introducing a web component.');
+  warn('Ensure the component style is extensible via `className`.');
 }
 
+// Ensure new native components are extensible by consumers.
 const nativeComponentIntroduced = createdFiles.some(filePath => (
   filePath.match(/native\/packages\/react-native-bpk-component.+\/src\/.+\.js/)
 ));
 
 if (nativeComponentIntroduced) {
-  message('It looks like you are introducing a native component');
-  warn('Ensure the component style is extensible via `style`');
+  message('It looks like you are introducing a native component.');
+  warn('Ensure the component style is extensible via `style`.');
```

Comment:
I feel like we could do some fancy checking on `propTypes` of the native component and break the build if style is not there, wdyt?

Reply:
We probably could, but I feel like that should be done in a further PR so as not to conflate this one